conans/client/cmd/export.py

```diff
@@ -86,27 +87,27 @@ def cmd_export(app, global_conf, conanfile_path, name, version, user, channel, g
     return ref, conanfile
 
 
-def _calc_revision(scoped_output, path, manifest, revision_mode):
+def _calc_revision(scoped_output, path, manifest, revision_mode, conanfile):
     if revision_mode not in ["scm", "scm_folder", "hash"]:
         raise ConanException("Revision mode should be one of 'hash' (default) or 'scm'")
 
     # Use the proper approach depending on 'revision_mode'
     if revision_mode == "hash":
         revision = manifest.summary_hash
     else:
-        f = '-- "."' if revision_mode == "scm_folder" else ""
+        # Exception to the rule that tools should only be used in recipes, this Git helper is ok
```

Comment:
This file modification should be a refactor to use ``Git()`` helper instead of duplicating here the functionality